plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1106,7 +1088,7 @@ def get_snmpV2cWrite_params(self, snmpV2cWriteDetails):
                 snmpV2cWrite.append(value)
         return snmpV2cWrite
 
-    def get_httpsRead_params(self, httpsReadDetails):
+    def get_httpsRead_params(self, https_read_details):
```

Comment:
get_https_read_params

Check the docstring

Reply:
done